extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -115,8 +120,10 @@ export class SchemaCompareResult {
 			this.createGenerateScriptButton(view);
 			this.createApplyButton(view);
 			this.createOptionsButton(view);
+			this.createOpenScmpButton(view);
 			this.createSourceAndTargetButtons(view);
 			this.resetButtons(false); // disable buttons because source and target aren't both selected yet
+			this.openScmpButton.enabled = true;
```

Comment:
why not from inside resetButtons()

Reply:
added an enum to pass to resetButtons() for the different scenarios